cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/SchedulesUtil.java

```diff
@@ -22,7 +22,7 @@
 /**
  *  Utility class for Scheduler.
  */
-public class Schedules {
+public class SchedulesUtil {
```

Comment:
Does this class have single method that is used in a single place? I think in that case it is better to keep that method in the class that uses it.